components/AppBar/src/MDCAppBarViewController.h

```diff
@@ -36,6 +36,14 @@
  */
 @property(nonatomic, strong, nonnull) MDCHeaderStackView *headerStackView;
 
+/**
+ When this flag is set to YES, the height of the app bar will be automatically adjusted to the sum
+ of the top bar height and the bottom bar height.
```

Comment:
Suggested docs:

```
Enabling this property will disable `minMaxHeightIncludesSafeArea` on the flexible header view.
```

Reply:
Added as additional docs in 1b97d553ff0a58d5597bbe7382a338085125677f.